.golangci.yml

```diff
@@ -21,7 +21,9 @@ linters-settings:
       # Replace these for trivy
       - github.com/docker/docker
       - oras.land/oras-go
-
+      # Replace for grom update
+      # TODO(chrisgacsal): remove when https://github.com/anchore/sqlite/pull/3 is merged
```

Comment:
Should we open an issue to keep track on it?

Reply:
Honestly, I'd not as I have no idea how long it will take the upstream project to get that PR merged and I'd avoid polluting the GH issues with TODOs. We should monitor the `TODO` comments in the code anyway and I think it is sufficient to check this next time when we bump the version of `gorm`. wdyt?

Reply:
https://github.com/openclarity/vmclarity/issues/382